README.md

```diff
@@ -2,15 +2,25 @@
 
 An open source version of Switchboard.
 
-***Join us to work In Real Life at noon every Tuesday [@RAIN](https://goo.gl/maps/NvamavVu2CtUMQFp9)***
+***Join us first and third Wednesday evenings for sprint wrap & plan, and each Tuesday evening for check-in. [EUG-Go](https://www.meetup.com/eug-go)***
```

Comment:
I wonder if we should mention that our meetings happen in convenient Pacific Time hours.